packages/manager/src/views/WebPropertyDetailPage/components/SSR/StaticDeployment.tsx

```diff
@@ -85,213 +96,294 @@ export const StaticDeployment = () => {
       }
     }
   };
+  const perPageOptions = [
+    { title: '5', value: 5 },
+    { title: '10', value: 10 },
+    { title: '15', value: 15 },
+    { title: '20', value: 20 },
+    { title: '50', value: 50 }
+  ];
 
+  const [page, setPage] = useState(1);
+  const [perPage, setPerPage] = useState(5);
+  const onPageSet = (_: any, pageNumber: number) => {
+    setPage(pageNumber);
+  };
+  const onPerPageSelect = (
+    _event: React.MouseEvent | React.KeyboardEvent | MouseEvent,
+    newPerPage: number,
+    newPage: number
+  ) => {
+    setPerPage(newPerPage);
+    setPage(newPage);
+  };
   return (
-    <div>
-      <p style={{ justifyContent: 'end', display: 'flex', fontSize: '14px' }}>
-        <Label color="gold" isCompact style={{ marginRight: '8px' }}>
-          <>
-            <SyncAltIcon /> &nbsp; Enabled
-          </>
-        </Label>
-        AutoSync Enabled for application
-      </p>
-      {!staticDeploymentData?.length ? (
-        <EmptyState>
-          <EmptyStateIcon icon={CubesIcon} />
-          <Title headingLevel="h4" size="lg">
-            No Static Deployment exists.
-          </Title>
-          <EmptyStateBody>Please create an deployment to view them here</EmptyStateBody>
-        </EmptyState>
-      ) : (
-        <TableComposable aria-label="spa-property-list">
-          <Caption>SPA&apos;s DEPLOYED</Caption>
-          <Thead noWrap>
-            <Tr>
-              <Th textCenter>SPA Name</Th>
-              <Th textCenter>Environments</Th>
-              <Th textCenter>Ref</Th>
-              <Th textCenter>Path</Th>
-              <Th textCenter>Internal Access URL</Th>
-              <Th textCenter>Router URL</Th>
-              <Th textCenter style={{ justifyContent: 'space-evenly', display: 'grid' }}>
-                Actions
-              </Th>
-            </Tr>
-          </Thead>
+    <>
+      <Split hasGutter className="pf-u-mb-md">
+        <SplitItem isFilled>
+          <Select
```

Comment:
@pjain24 Can you try writing a separate component EnvFilter for this , and call it in containerized and static, as code is being repeated in both the files.
Create a component that accepts data and returns a dropdown